### PR TITLE
Functions/discord int fix

### PR DESCRIPTION
**Pull request by chrismclarke**
Fix discord integration
- change `request` package for `axios` (easier to handle response to post/get request)
- fix variable bindings/typos
- tidy code

Tested for updating map pin, howto and event on dev. config variable has been set on prod so should start working there also once in.

image

image

image

### CODE REVIEW

**Comment by davehakkens** [comment]
looking neaat.. cant wait to use this one!
Is the Slack webhook also working?

**Comment by chrismclarke** [comment]
I see plenty coming through the slack webhook so assume it's working as intended. Although if anybody notices any issues feel free to flag up and I'll take a look.

**Comment by davehakkens** [comment]
huh into which channel? 
Because in the #community-notifications-dev I don't see any updates.
And in the #community-notifications i see updates. But I think they are still the old ones as it shows the old text version without the map url Location here (soon our own map)

**Comment by chrismclarke** [comment]
Oh, I'm with you now... hmm, strange, can add to the list of things to investigate.

File: functions/src/Integrations/firebase-discord.ts [hunk 1]
```diff
@@ -9,89 +9,61 @@ const DISCORD_WEBHOOK_URL = CONFIG.integrations.discord_webhook
...
-    const slug = info.slug
-    request.post(
-      DISCORD_WEBHOOK_URL,
-      {
-        json: {
-          content: `📓 Yeah! New How To "* ${title} *" by ${user}
+    const { _createdBy, title, slug } = info
+    await axios
+      .post(DISCORD_WEBHOOK_URL, {
+        content: `📓 Yeah! New How To "* ${title} *" by ${_createdBy}
```
Comment (BenGamma) [inline functions/src/Integrations/firebase-discord.ts]: I think here the message would be : `📓 Yeah! New How To **${title}** by *${_createdBy}*` without the " and with the correct font style markdown (**bold** with ** around how to title and _italic_ for creator name with one *)

File: functions/src/Integrations/firebase-discord.ts [hunk 2]
```diff
@@ -9,89 +9,61 @@ const DISCORD_WEBHOOK_URL = CONFIG.integrations.discord_webhook
...
     const location = info.location.country
-    console.info(info)
-    request.post(
-      DISCORD_WEBHOOK_URL,
-      {
-        json: {
-          content: `📅 Jeej new event in *${location}* by ${user} posted here:
+    await axios
+      .post(DISCORD_WEBHOOK_URL, {
+        content: `📅 Jeej new event in *${location}* by ${user} posted here:
```
Comment (BenGamma) [inline functions/src/Integrations/firebase-discord.ts]: Same here for font style : `📅 Jeej new event in **${location}** by *${user}* posted here:`

**Comment by BenGamma** [comment]
@chrismclarke shoudl I create a new PR for the text updates or you are going to push some updates anyway on this branch ?

**Comment by chrismclarke** [comment]
This is already merged so I suggest updates go in a different PR. At the same time (or separately) you might want to make the required updates to the slack hook

**Pull request merged**